src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -404,6 +404,95 @@ protected virtual bool CheckInvocation(TexlNode[] args, DType[] argTypes, IError
             return CheckInvocationCore(args, argTypes, errors, out returnType, out nodeToCoercedTypeMap);
         }
 
+        #region CheckInvocation Replacement Project
+        public bool HandleCheckInvocation(TexlBinding binding, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        {
+            if (CheckTypesAndSemanticsOnly)
+            {
+                Contracts.Assert(!CompareLegacyCheckInvocation);
+
+                var result = CheckTypes(binding.CheckTypesContext, args, argTypes, errors, out returnType, out nodeToCoercedTypeMap);
+                CheckSemantics(binding, args, argTypes, errors, ref nodeToCoercedTypeMap);
+                return result;
+            }
+
+            if (CompareLegacyCheckInvocation)
```

Comment:
Should we add `#if DEBUG`, make sure this doesn't wind up in shipped code?

Reply:
I will add a test in PAClient